projects/1.20/assets/hexcasting/hexcasting/lang/zh_cn.flatten.json5

```
@@ -0,0 +1,2201 @@
+        one: "在栈下标为%d处获取到意外iota：%s",
+        many: "在栈下标为%2$d到%3$d处获取到意外iota：%4$s",
+      },
+
+      no_akashic_record: "%s处无阿卡夏记录",
+      disallowed: "已被服务器管理员禁用于施法",
+      disallowed_circle: "已被服务器管理员禁用于法术环",
+      invalid_spell_datum_type: "尝试将某无效类型的值用作SpellDatum：%s（class %s）。这是模组中的bug。",
+      unknown: "抛出异常（%s）。这是模组中的bug。",
+      stack_size: "超出了栈的大小上限",
+      bad_caster: "试图运行的图案需要强大的意识才能承受",
+
+      invalid_value: {
+        "": "本应在栈下标为%2$s处接受%1$s，而实际接受了%3$s",
+
+        class: {
+          double: "一个数",
+          boolean: "一个布尔值",
+          vector: "一个向量",
+          list: "一个列表",
+          widget: "一个虚指",
+          pattern: "一个图案",
+
+          entity: {
+            "": "一个实体",
+            item: "一个物品实体",
+            player: "一个玩家",
+            villager: "一个村民",
+            living: "一个生物",
+          },
+
+          entity_or_vector: "一个实体或向量",
+
+          unknown: "（未知，这是个bug）",
+        },
+
+        numvec: "一个数或向量",
+        numlist: "一个整数或列表",
+        "list.pattern": "一个由图案组成的列表",
+
+        double: {
+          positive: {
+            "": "一个正数",
+            less: "一个小于%d的正数",
+            "less.equal": "一个小于等于%d的正数",
+          },
+          between: "一个介于%d和%d之间的数",
+        },
+
+        int: {
+          "": "一个整数",
+          positive: {
+            "": "一个正整数",
+            less: "一个小于%d的正整数",
+            "less.equal": "一个小于等于%d的正整数",
+          },
+          between: "一个介于%d和%d之间的整数",
+        },
+
+        evaluatable: "可运行的事物",
+        bool_commute: "一个布尔值、0或1",
+      },
+
+      location_: {
+        too_far: "%s超出影响范围",
+        out_of_world: "%s不在此世界内",
+        too_close_to_out: "%s离世界边界太近了",
+        forbidden: "%s并未对你开放",
+        bad_dimension: "无法在此维度内传送",
+      },
+
+      bad_item: {
+        "": "需要%s，而实际持有%d个%s",
+        offhand: "需要在另一只手里持有%s，而实际持有%d个%s",
+
+        iota: {
+          "": "一个可以存储iota的地方",
+          read: "一个可以读出iota的地方",
+          write: "一个可以写入iota的地方",
+          readonly: "一个能够接受%s的地方",
+        },
+
+        media: "含有媒质的物品",
+        media_for_battery: "天然含有媒质的物品",
+        only_one: "仅一个物品",
+        eraseable: "一个可清除的物品",
+        bottle: "一个玻璃瓶",
+        rechargable: "一个可重新充能的物品",
+        colorizer: "一个染色剂",
+        variant: "一个有变种的物品"
+      },
+
+      bad_block: {
+        "": "本应在%2$s处接受%1$s，而实际接受了%3$s",
+        sapling: "一个树苗",
+        replaceable: "一个可放置方块的地方",
+      },
+
+      "circle.bool_directrix": {
+        no_bool: "%s处的iota实际为%s，而非布尔值",
+        empty_stack: "%s处的栈为空栈"
+      },
+    },
+    // ^ mishap
+
+    circles: {
+      no_exit: "%s处的媒质流无法找到出口",
+      many_exits: "%s处的媒质流的可选去路过多"
+    },
+
+
+    // === Patchi stuff! ===
+
+
+    landing: "我似乎发现了一种全新的魔法，一种在六边形网格上画各式奇诡图案的魔法。\
+      它深深地迷住了我。我决定着手撰写一本笔记，以此记述我的想法和研究成果。\
+      $(br2)$(l:https://forum.petra-k.at/index.php)论坛链接/$",
+
+    category: {
+      basics: {
+        "": "初入咒法",
+        desc: "研究这种魔法的人士会用$(l:items/staff)$(item)法杖/$在空中绘制奇异的图案来施放他们口中的“$(hex)咒术/$”，\

```

```suggestion
        desc: "研究这种魔法的人士会用$(l:items/staff)$(item)法杖/$在空中绘制奇异的图案来施放他们口中的“_咒术”，\
```
原文的所有`$(hex)XXX/$`貌似都是`_XXX`（好像是用空格结束？）形式的

---

这里是帕秋莉的宏，这两个展开之后的效果是一样的

---

OK